pyproject.toml

```
@@ -138,9 +140,17 @@ lint.allowed-confusables = [
 lint.ignore = [
     "COM812", # missing-trailing-comma (ruff-format recommended)
     "D203", # one-blank-line-before-class
+    "D205",
     "D212", # blank-line-before-class
+    "D401",
+    "EM101",
+    "EM102",
```

EM101 and EM102 are definitely useful. I would prefer not to ignore them.
```suggestion
    "D401",  # non-imperative-mood
```

---

```suggestion
    "D401",  # non-imperative-mood
    "EM101",
    "EM102",
```

See the comment I wrote below: I think this not great. If I see a stack trace, I want to see that the error message actually came from the line that raised, and wasn't forwarded from a caught exception or similar.